modules/fx/arrow-fx/src/main/kotlin/arrow/fx/typeclasses/Concurrent.kt

```diff
@@ -973,15 +973,12 @@ interface Concurrent<F> : Async<F> {
 }
 
 interface ConcurrentFx<F> : AsyncFx<F> {
-  val concurrent: Concurrent<F>
-
-  override val async: Async<F>
-    get() = concurrent
-
-  fun <A> concurrent(c: suspend ConcurrentSyntax<F>.() -> A): Kind<F, A> {
-    val continuation = ConcurrentContinuation<F, A>(concurrent)
+  override val M: Concurrent<F>
```

Comment:
If you're not overriding it, is it necessary?

Reply:
It is in the implementation above, what i'm doing here is just narrowing the type to be concurrent